openverse_catalog/dags/providers/provider_reingestion_workflows.py

```diff
@@ -61,7 +61,7 @@ def __post_init__(self):
         # 60 total reingestion days
         ingester_class=EuropeanaDataIngester,
         max_active_tasks=3,
-        pull_timeout=timedelta(hours=12),
+        pull_timeout=timedelta(hours=16),
```

Comment:
This is the only change we need to solve issue 954. Could you undo the ones on the rest of the files? Thanks!